test/environment/cleanup.go

```diff
@@ -0,0 +1,40 @@
+package environment
+
+import (
+	"fmt"
+
+	"github.com/alibaba/pouch/client"
+	"github.com/pkg/errors"
+)
+
+// PruneAllImages deletes all images from pouchd.
+func PruneAllImages(apiClient client.ImageAPIClient) error {
```

Comment:
Why `func PruneAllImages(apiClient client.ImageAPIClient) error`?
Not `func (apiClient client.ImageAPIClient) PruneAllImages() error`?

Reply:
Original idea is that the `PruneAllImages` is only used in test package. It comes from `docker  system prune`. If you need this in pouch, we can implement it in `cli` package.

Reply:
It is not in pouch's first stage.
Maybe in the future, we will support this.

Reply:
Got it. Make senses